custom.conf

```diff
@@ -0,0 +1,24 @@
+    tcp_nopush on;
+    tcp_nodelay on;      
+
+    types_hash_max_size 2048;
+    server_tokens off;
+    client_max_body_size 200M;
+    client_body_buffer_size 128k;
+    server_names_hash_bucket_size 64;
+    map_hash_bucket_size 512;
+    proxy_buffer_size   512k;
+    proxy_buffers   4 512k;
+    proxy_busy_buffers_size   512k;
+
+    ##
+    # Gzip Settings
+    ##
+    gzip on;
```

Comment:
this one would be nice if we make it configurable, you can see how we hande other confs:

'''
for filename in /etc/nginx/conf.d/*.conf; do
  if [ -e "${filename}" ] ; then
    cp ${filename} ${filename}.tmp
    envsubst '${PHP_HOST} ${PHP_PORT} ${NGINX_DEFAULT_SERVER_NAME} ${NGINX_DEFAULT_ROOT} ${NGINX_SUBFOLDER} ${NGINX_SUBFOLDER_ESCAPED} ${NGINX_OSB_BUCKET} ${NGINX_OSB_RESOLVER} ${DRUPAL_PUBLIC_FILES_PATH} ${NGINX_CACHE_CONTROL_HEADER}' < $filename.tmp > $filename
    rm ${filename}.tmp
  fi
done
```
In this case should be:

```
for filename in /etc/nginx/conf.d//*.conf; do
  if [ -e "${filename}" ] ; then
    cp ${filename} ${filename}.tmp
    envsubst '${PHP_HOST} ${PHP_PORT} ${NGINX_DEFAULT_SERVER_NAME} ${NGINX_DEFAULT_ROOT} ${NGINX_SUBFOLDER} ${NGINX_SUBFOLDER_ESCAPED} ${NGINX_OSB_BUCKET} ${NGINX_OSB_RESOLVER} ${DRUPAL_PUBLIC_FILES_PATH} ${NGINX_CACHE_CONTROL_HEADER}' < $filename.tmp > $filename
    rm ${filename}.tmp
  fi
done
```
We should just add a new env variable 'export NGINX_GZIP_ENABLE=${NGINX_GZIP_ENABLE:-1}` with a default value of 1.